keys-and-keywords/ubuntu-keywords.robot

```diff
@@ -40,3 +40,22 @@ Install Package
     Write Into Terminal    apt-get install --assume-yes ${package}
     Read From Terminal Until Prompt
     Set DUT Response Timeout    30s
+
+Stop Logging To Terminal
+    [Documentation]    This keyword stops all unwanted logging to the terminal
+    ...    we are using for the shell. This prevents garbage from being logged.
+    Set Logging Level    0
+
+Get Logging Level
+    [Documentation]    This keyword returns TRUE if logging is disabled and
+    ...    FALSE if it is not.
+    ${ret}=    Execute Linux Command    cat /proc/sys/kernel/printk | cut -f1
```

Comment:
`cut -f 1 /proc/sys/kernel/printk` will also do

cat is redundant in many cases like this.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/compare/4f351632f30b20656555653ad0d6b40a726bb3be..c2910d941d4d01840e0e8479d6c1dbaa8eb8f728#diff-b7afbf080f275d1995a5212ce98c26d58fb1004705c786cc31496e1043ac04b6L52